SoftLayer/managers/network.py

```diff
@@ -470,47 +470,15 @@ def list_global_ips(self, version=None, identifier=None, **kwargs):
         kwargs['filter'] = _filter.to_dict()
         return self.account.getGlobalIpRecords(**kwargs)
 
-    def list_subnets(self, identifier=None, datacenter=None, version=0,
-                     subnet_type=None, network_space=None, **kwargs):
+    def list_subnets(self, **kwargs):
         """Display a list of all subnets on the account.
 
         This provides a quick overview of all subnets including information
         about data center residence and the number of devices attached.
-
-        :param string identifier: If specified, the list will only contain the
-                                    subnet matching this network identifier.
-        :param string datacenter: If specified, the list will only contain
-                                    subnets in the specified data center.
-        :param int version: Only returns subnets of this version (4 or 6).
-        :param string subnet_type: If specified, it will only returns subnets
-                                     of this type.
-        :param string network_space: If specified, it will only returns subnets
-                                       with the given address space label.
-        :param dict \\*\\*kwargs: response-level options (mask, limit, etc.)
         """
         if 'mask' not in kwargs:
             kwargs['mask'] = DEFAULT_SUBNET_MASK
 
-        _filter = utils.NestedDict(kwargs.get('filter') or {})
-
-        if identifier:
-            _filter['subnets']['networkIdentifier'] = (
-                utils.query_filter(identifier))
-        if datacenter:
-            _filter['subnets']['datacenter']['name'] = (
-                utils.query_filter(datacenter))
-        if version:
-            _filter['subnets']['version'] = utils.query_filter(version)
-        if subnet_type:
-            _filter['subnets']['subnetType'] = utils.query_filter(subnet_type)
-        else:
-            # This filters out global IPs from the subnet listing.
-            _filter['subnets']['subnetType'] = {'operation': '!= GLOBAL_IP'}
```

Comment:
This is the only section I wanted removed. We still need to be able to filter subnets on other fields.